kalliope/neurons/ansible_playbook/ansible_playbook.py

```diff
@@ -23,25 +26,47 @@ def __init__(self, **kwargs):
 
         # check if parameters have been provided
         if self._is_parameters_ok():
+            options = self._get_options()
 
-            variable_manager = VariableManager()
+            # initialize needed objects
             loader = DataLoader()
-            options = self._get_options()
+
             passwords = {'become_pass': self.sudo_password}
 
-            inventory = Inventory(loader=loader, variable_manager=variable_manager, host_list="localhost")
-            variable_manager.set_inventory(inventory)
-            playbooks = [self.task_file]
+            inventory = InventoryManager(loader=loader, sources="localhost,")
 
-            executor = PlaybookExecutor(
-                playbooks=playbooks,
-                inventory=inventory,
-                variable_manager=variable_manager,
-                loader=loader,
-                options=options,
-                passwords=passwords)
+            # variable manager takes care of merging all the different sources to give you a unified
+            # view of variables available in each context
+            variable_manager = VariableManager(loader=loader, inventory=inventory)
+            variable_manager.set_inventory(inventory)
 
-            executor.run()
+            playbooks = None
+            with open(self.task_file, 'r') as stream:
+                try:
+                    playbooks = yaml.load(stream)
+                except yaml.YAMLError as exc:
+                    print(exc)
```

Comment:
Debug ? or more explicit message ?

Reply:
I just print the error so the user know why it failed in case of yaml syntax.
Should I wrap it with a header or something?

Reply:
Hum it is not a big deal anyway, you can leave it like this